packages/material-ui-lab/src/Container/Container.js

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { withStyles } from '@material-ui/core/styles';
+import { capitalize } from '@material-ui/core/utils/helpers';
+
+const styles = theme => ({
+  root: {
+    width: '100%',
+    marginLeft: 'auto',
+    boxSizing: 'border-box',
+    marginRight: 'auto',
+    paddingLeft: theme.spacing(2),
+    paddingRight: theme.spacing(2),
+    [theme.breakpoints.up('sm')]: {
+      paddingLeft: theme.spacing(3),
+      paddingRight: theme.spacing(3),
+    },
+  },
+  fixed: Object.keys(theme.breakpoints.values).reduce((acc, breakpoint) => {
+    const value = theme.breakpoints.values[breakpoint];
+
+    if (value !== 0) {
+      acc[theme.breakpoints.up(breakpoint)] = {
+        maxWidth: value + theme.spacing(3) * 2,
+      };
+    }
+    return acc;
+  }, {}),
+  maxWidthXs: {
+    [theme.breakpoints.up('xs')]: {
+      maxWidth: Math.max(theme.breakpoints.values.xs, 396) + theme.spacing(2) * 2,
```

Comment:
Just curious, I'm sure I'm missing something obvious, why 396?

Reply:
Because 400 / 12 = 33.33333, but you are right, it's a topic for another pull request, I'm reverting.

Reply:
The best answer I can give is that it feels good :/.